torch/csrc/jit/script/sugared_value.cpp

```diff
@@ -289,15 +289,32 @@ Value* SimpleValue::len(const SourceRange& loc, Function& m) {
   }
 }
 
+std::shared_ptr<SugaredValue> callClassMethod(
+    const ClassTypePtr& class_ptr,
+    const std::string& desugared_name,
+    const SourceRange& loc,
+    Function& m,
+    at::ArrayRef<NamedValue> inputs,
+    at::ArrayRef<NamedValue> attributes,
+    size_t n_binders) {
+  if (!class_ptr->getMethod(desugared_name)) {
+    throw ErrorReport(loc) << class_ptr->python_str() << " does not define a "
+                           << desugared_name << " method";
+  }
+
+  Value* self = inputs[0].value(*m.graph());
```

Comment:
maybe assert that `inputs.size() == 1` in case this gets called from somewhere else in the future